.github/workflows/gitlab-ci.yml

```diff
@@ -0,0 +1,19 @@
+name: Mirror and run GitLab CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Mirror + trigger CI
+      uses: SvanBoxel/gitlab-mirror-and-ci-action@master
+      with:
+        args: "https://gitlab.com/ailisp/nearcore"
```

Comment:
should this be nearprotocol/nearcore?

Reply:
This was mean to experiment with fork first to avoid break origin.